components/AuthorInfo.js

```diff
@@ -1,5 +1,23 @@
 import { renderRichText } from "../lib/richText";
-import { getAuthorInfo } from "../lib/api";
 
-export default function AuthorInfo({ AuthorInfoData }) {
+export default function AuthorInfo({ authorInfo }) {
+
+    const {
+        additional_information: cameraInfo,
+        author_name: authorName, 
+        signature,
+        author_image: authorImage
+    } = authorInfo;
+
+    return (
+        <>
+        {renderRichText(authorName)}
+        {renderRichText(cameraInfo)}
+        <img src={signature.url} alt="" />
+        <a>Signature dimensions: {signature.dimensions.width} x {signature.dimensions.height} </a>
+        <br /> <br />
+        <img src={authorImage.url} alt="" />
+        <a>Author image dimensions: {authorImage.dimensions.width} x {authorImage.dimensions.height} </a>
```

Comment:
Styling comment: let's indent these lines in one level from the <> </>

Reply:
Follow up here, creating a .prettier config may help!

Reply:
Thanks, just fixed it! And will look into a .prettier config this week!